docker-compose.yml

```diff
@@ -34,7 +41,10 @@ services:
   redis:
     container_name: protectedplanet-redis
     image: redis
-    network_mode: host
+    ports:
+      - "6379:6379"
```

Comment:
Please change to "6379" (without host port), so it won't conflict with host port. This is Jenkins test, no one will connect from the host.